src/DynamoCoreWpf/Properties/AssemblyInfo.cs

```diff
@@ -35,3 +35,4 @@
 [assembly: InternalsVisibleTo("ViewExtensionLibraryTests")]
 [assembly: InternalsVisibleTo("DynamoWPFCLI")]
 [assembly: InternalsVisibleTo("CommandLineTests")]
+[assembly: InternalsVisibleTo("Watch3DNodeModelsWpf")]
```

Comment:
Why is this needed? Are you exposing internals from `DynamoCoreWpf` in `Watch3DNodeModelsWpf`?

Reply:
Yes, I changed `UnregisterNodeEventHandlers` from protected to internal so now this is needed because multiple other libraries need access after this PR